homeassistant/components/zha/select.py

```diff
@@ -73,6 +73,8 @@ class ZHAEnumSelectEntity(ZhaEntity, SelectEntity):
     _attr_entity_category = EntityCategory.CONFIG
     _attribute_name: str
     _enum: type[Enum]
+    # needs fixing: Change all enums to uncapitalized snake_case
```

Comment:
IMO, the enums should stay as-is. I don't think it's the right call to all have them completely lower case.
If we decide to change this, we'd need to change enums in HA, quirks, and zigpy too.

I think lower-casing (like you currently do) for the translation keys is fine.

Reply:
Okay.